### DISCUSSION TITLE
Move development to Replit?

### DISCUSSION THREAD

**Discussion by aidan-crockett** [theforumhelpers/theforumhelpers.github.io#251] [General]
Replit is great, because you can really simplify your relative urls and use `/`

(I'm not suggesting we use replit for all hosting, just to do some development)

**Comment by aidan-crockett** [comment]
bump, to give anyone a notification

**Comment by BarelySmooth** [comment]
I think the current system works fine. Moving development to replit means asking contributors to create one more account. I also think its possible for individual contributors to contribute to the repo from replit instead of moving the entire development there.

**Reply by aidan-crockett** [reply]
I believe you can connect your replit account to your github account, making things a bit less painful to setup, but I appreciate your point. Doing some development instead of all of it might be a good idea

**Comment by Shluffy** [comment]
I don't have a replit account. While I can make one, I think that github is a good place. And `*.github.io` sounds better then `*.repl.co` imo

**Reply by aidan-crockett** [reply]
> I don't have a replit account. While I can make one, I think that github is a good place. And `*.github.io` sounds better then `*.repl.co` imo

you misunderstand - the development happens on replit but the code is ultimately pushed to github

**Reply by Accio1** [reply]
I think GitHub has several important features, like Issues, Pull Requests, Git/GitHub desktop, discussions, teams, etc. that it would be better to remain on GitHub rather than replit.

**Reply by Shluffy** [reply]
> > I don't have a replit account. While I can make one, I think that github is a good place. And `*.github.io` sounds better then `*.repl.co` imo
> 
> you misunderstand - the development happens on replit but the code is ultimately pushed to github

I am not very good with replit, never done it

**Comment by Steve0Greatness** [comment]
people can use replit if they want to, I just don't think you should need one to contribute(like whenflagclicked.org)

**Comment by leahcimto** [comment]
Another thing to keep in mind, replit can't be linked on Scratch for unmoderated chat reasons.

**Reply by Quantum-Codes** [reply]
Isn't github also unmoderated? Like this place

**Reply by leahcimto** [reply]
Scratch's rules, not mine.

**Reply by Quantum-Codes** [reply]
Lol xDDD

**Reply by aidan-crockett** [reply]
> Isn't github also unmoderated? Like this place

Github is considered to be a "creative" site like Youtube, so the Scratch Team tolerates it.

**Reply by aidan-crockett** [reply]
> Another thing to keep in mind, replit can't be linked on Scratch for unmoderated chat reasons.

In the past, I've been muted for linking *.repl.co but idk if replit itself <sub><sup>(replitself 😆)</sup></sub> is blocked

**Reply by Quantum-Codes** [reply]
Me too got muted for same reason lol
For 5 mins

**Reply by leahcimto** [reply]
src https://scratch.mit.edu/discuss/topic/447454/?page=1#post-4502371

**Reply by Quantum-Codes** [reply]
I too saw that.  Honestly I think ST messed up.

Repl.co is censored. But sharing user websites is ok.
Repl.it isn't censored but it has **live chat**. 
I am ok with this mistake tho.., xD I usually ask doubts using repl. it rather than repl.co lol
